Mention the bug bounty program (#207)
Mbed-TLS/mbedtls-docs: project/vulnerabilities.md

## Reporting a vulnerability

> If you think you have found a security vulnerability in Mbed TLS (including TF-PSA-Crypto), then please send an email to the security team at <mailto:mbed-tls-security@lists.trustedfirmware.org>. For more information on the reporting and disclosure process, please see the [TrustedFirmware.org security incident handling process](https://trusted-firmware-docs.readthedocs.io/en/latest/security_center/incident_handling_process.html). There are some caveats to that process when applied to Mbed TLS, as listed below:

Reviewer: ```suggestion
If you think you have found a security vulnerability in Mbed TLS or TF-PSA-Crypto, then please send an email to the security team at <mailto:mbed-tls-security@lists.trustedfirmware.org>. For more information on the reporting and disclosure process, please see the [TrustedFirmware.org security incident handling process](https://trusted-firmware-docs.readthedocs.io/en/latest/security_center/incident_handling_process.html). There are some caveats to that process when applied to Mbed TLS, as listed below:
``` ?

- Mbed TLS currently does not have any registered ESSes and so there is no primary embargo period.
-  Mbed TLS contains strong cryptography software and to comply with export control restrictions, must only distribute software publicly. As a result, security fixes cannot be shared privately with Trusted Stakeholders, although other vulnerability information can be.
- The nature of Mbed TLS often means that security fixes reveal enough information for a skilled attacker to re-construct the originally reported exploit. This combined with the previous caveat means we often expect to have to withhold security fixes until the public disclosure date.
- Mbed TLS is subject to a lot of scrutiny by security researchers who often have their own disclosure timelines when reporting vulnerabilities. As a result, the default 90 days public embargo period may often not apply.

> ## Bug bounty program

Reviewer 2: If you are after a bug bounty, it's unclear from this document whether you should also report to the security alias — we prefer that you should pick one, so we should state that. I think we should give a short guide about which one to pick. The salient differences as I see them:

* Only the bug bounty gives out money.
* If you go through the bug bounty program, you abandon control over disclosure.
* The bug bounty program requires a PoC. Reporting to the alias lets you choose your own way of appearing to be reliable.
* Reports through the bug bounty program have an intake delay. The alias reaches the developers directly.

Author: Thanks for the suggestion! That's a lot more helpful than the "please read the terms and conditions" I had in mind :)

> Mbed TLS (including TF-PSA-Crypo) is part of Arm's Trusted Firmware [bug bounty program](https://app.intigriti.com/programs/arm/trustedfirmware/detail).

Reviewer: ```suggestion
Mbed TLS and TF-PSA-Crypto are part of Arm's Trusted Firmware [bug bounty program](https://app.intigriti.com/programs/arm/trustedfirmware/detail).
``` ?
